### PR TITLE
Always remember me

### LINKED ISSUE #8004: Remove »remember log in« and just remember automatically
On the Login page the layout it the following:

User, Password, Submit, Remember me.

I think this is a pretty weird layout especially as the most common one being first the user and password, then remember and AFTER THAT the submit button.

this gets majorly annoying when someone goes in with keyboard and, just out of muscle memory goes User, tab, password, tab, spacebar (to select remember me), enter, which instead makes the spacebar trigger the submit and not the checkbox, has happened to me already on multiple instances.

so it would be awesome to just swap those 2, or at least their tab order.

### PR DESCRIPTION
**Pull request by rullzer**
Fixes #8004

Remove remember me from the login page and just always remember.

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: core/Controller/LoginController.php [hunk 1]
```diff
@@ -240,7 +238,7 @@ private function generateRedirect($redirectUrl) {
 	 * @param string $timezone_offset
 	 * @return RedirectResponse
 	 */
-	public function tryLogin($user, $password, $redirect_url, $remember_login = false, $timezone = '', $timezone_offset = '') {
+	public function tryLogin($user, $password, $redirect_url, $remember_login = true, $timezone = '', $timezone_offset = '') {
```
Comment (MorrisJobke) [inline core/Controller/LoginController.php]: Is the parameter still needed?

Comment (rullzer) [reply core/Controller/LoginController.php]: It is used in a few places. We could get rid of it. But I want to do baby steps

**Review by jancborchardt — APPROVED** [review]
Good stuff, finally! :)

**Review by juliusknorr — CHANGES_REQUESTED** [review]
One nitpick, besides that :+1:

**Pull request merged**